examples/api/lib/material/card/card.2.dart

```diff
@@ -13,98 +13,34 @@ void main() {
 class CardExamplesApp extends StatelessWidget {
   const CardExamplesApp({super.key});
 
-  @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      theme: ThemeData(colorSchemeSeed: const Color(0xff6750a4), useMaterial3: true),
-      home: Scaffold(
-        appBar: AppBar(title: const Text('Card Examples')),
-        body: const Column(
-          children: <Widget>[
-            Spacer(),
-            ElevatedCardExample(),
-            FilledCardExample(),
-            OutlinedCardExample(),
-            Spacer(),
-          ],
-        ),
-      ),
+  Widget buildContent(String cardName) {
```

Comment:
We should really encourage factoring into widgets rather than build methods. A _SampleCard widget with a const constructor would do the job here.